Userland/Libraries/LibWeb/Layout/Box.h

```diff
@@ -121,6 +121,16 @@ class Box : public NodeWithStyleAndBoxModelMetrics {
 
     virtual float width_of_logical_containing_block() const;
 
+    struct BorderRadiusData {
+    public:
+        int top_left;
+        int top_right;
+        int bottom_right;
+        int bottom_left;
```

Comment:
Should these not be `float`?
Either way, please explicitly initialize primitive variables :)

Reply:
I don't really know what we should do with fractional pixel values though... For now, I would keep it as int and add a fixme.